tests/test_hermes_doctor_bank_routing.py

```
@@ -0,0 +1,361 @@
+            no_fix=True,
+        )
+        cli_mod.mnemosyne_command(args)
+        out = capsys.readouterr().out
+        assert "resolved_bank: reverse-engineer" in out
+        assert "resolved_db:" in out
+        assert str(db_path) in out
+
+    def test_stats_sleep_routing_unaffected(self, tmp_path, monkeypatch):
+        """Existing stats/sleep routing not broken."""
+        home = tmp_path / "profiles" / "work"
+        _write_config(home, True)
```

Ruff (0.15.20)

[warning] Boolean positional value in function call

(FBT003)